daktari/checks/etc_hosts.py

```diff
@@ -0,0 +1,25 @@
+from daktari.check import Check, CheckResult
+from daktari.os import OS
+
+# We found that if /etc/hosts used a mixture of tabs and spaces
+# it caused some applications to ignore /etc/hosts and perform DNS lookups instead.
+# This check enforces a single space in any blankspace in /etc/hosts.
+
+
+class EtcHostsFormattedCorrectly(Check):
```

Comment:
is it an awful idea that this suggests the correct formatting, or formats it for them 😂

Reply:
Oh i guess your sudo command does it for them. Nice. Ignore.